### PR TITLE
Various Jscs fixes

### PR DESCRIPTION
**Pull request by paulrouget**
Depends on #336

### CODE REVIEW

File: src/browser/github.js [hunk 1]
```diff
@@ -44,14 +44,14 @@ define((require, exports, module) => {
...
         console.log('Github:', response.status);
       }
       console.log(`Github: pulling in ${interval}ms`);
       timeout = setTimeout(() => pull(resolve), interval);
     }).catch(error => {
-      console.error("Github: fetch error", error);
+      console.error('Github: fetch error', error);
       console.log(`Github: pulling in ${interval}ms`);
       timeout = setTimeout(() => pull(resolve), interval);
     });
```
Comment (Gozala) [inline src/browser/github.js]: This seems unrelated

File: src/browser/iframe.js [hunk 2]
```diff
@@ -77,7 +77,7 @@ define((require, exports, module) => {
     onMetaChange: Event('mozbrowsermetachange'),
     onLocationChange: VirtualEvent((target, dispatch) => {
       target.addEventListener('mozbrowserlocationchange', event => {
-        target.setAttribute("uri", event.detail);
+        target.setAttribute('uri', event.detail);
         dispatch(event);
       });
     }),
```
Comment (Gozala) [inline src/browser/iframe.js]: Unrelated changes.

File: src/lang/functional.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ define((require, exports, module) => {
     return (...args) => {
       let index = lambdas.length;
       let result = lambdas[--index](...args);
-      while (0 <= --index) {
```
Comment (gordonbrander) [inline src/lang/functional.js]: Nit: lots of "noise" in a big commit. It makes it harder to review.

File: src/common/url-helper.js [hunk 4]
```diff
@@ -49,7 +49,7 @@ define((require, exports, module) => {
     try {
       new URL(str);
       return false;
-    } catch(e) {
```
Comment (gordonbrander) [inline src/common/url-helper.js]: Noise

File: src/common/url-helper.js [hunk 5]
```diff
@@ -17,7 +17,7 @@ define((require, exports, module) => {
   const isAboutURL = url => {
     try {
       return new URL(url).protocol == 'about:';
-    } catch(e) {
```
Comment (gordonbrander) [inline src/common/url-helper.js]: Noise

File: src/common/keyboard.js [hunk 6]
```diff
@@ -42,20 +42,20 @@ define((require, exports, module) => {
   });
 
   const readChord = input =>
-    input.trim().
```
Comment (gordonbrander) [inline src/common/keyboard.js]: Moar noise

**Comment by gordonbrander** [comment]
It looks like all the commits but df7feaa were functional changes (unrelated to Jscs), whereas df7feaa was a code style commit.

**Comment by gordonbrander** [comment]
Wait nvm, this is a rebase problem I think.

**Comment by gordonbrander** [comment]
Yup this is good.
